python/vineyard/deploy/local.py

```diff
@@ -134,7 +142,7 @@ def start_vineyardd(etcd_endpoints=None,
             raise RuntimeError('vineyardd exited unexpectedly with code %d, error is:\n%s' % (rc, err))
 
         logger.debug('vineyardd is ready.............')
-        yield proc, socket
+        yield proc, socket, etcd_endpoints
```

Comment:
Please

1. check all use of `vineyard.deploy.local.start_vineyardd`  in the repo (including your previous notebook) to ensure that nothing is broken by this change
2. revisit the implementation of `vineyard.deploy.distributed.start_vineyardd` to ensure the function signatures are as consistent as possible.